.github/workflows/clean_install_examples.yml

```diff
@@ -0,0 +1,28 @@
+# This workflow checks that examples can be installed cleanly.
+# It only runs on PRs that affect the `examples/` directory.
+
+name: Check examples install status
+
+on: 
+  pull_request:
+    paths:
+      - 'examples/**'
```

Comment:
Can we update the dependabot workflow to also do this but for plugins?

Reply:
this already happens for plugins because doing a `npm ci` is part of the workflow for running the plugin tests

Reply:
Sorry I meant can we make the "fixup the package locks" workflow only run on PRs that are from dependabot and touch files in plugins?

Reply:
oh yeah will do